--- a/CMakeLists.txt	2020-05-21 16:12:59.355389456 +0200
+++ b/CMakeLists.txt	2020-05-21 16:15:24.649388902 +0200
@@ -188,40 +188,41 @@
 endif()
 message("cmake install path  ${CMAKE_INSTALL_PREFIX}")
 
-install( TARGETS amdvdi_static ARCHIVE DESTINATION lib LIBRARY DESTINATION lib COMPONENT applications)
+install( TARGETS amdvdi_static EXPORT amdvdi_export ARCHIVE DESTINATION lib64 LIBRARY DESTINATION lib64 COMPONENT applications)
+INSTALL(EXPORT amdvdi_export DESTINATION lib64)
 FILE(GLOB headerfiles "${CMAKE_CURRENT_SOURCE_DIR}/*.h*")
 INSTALL (
     FILES ${headerfiles}
-    DESTINATION include)
+    DESTINATION include/rocclr)
 INSTALL (
    DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/include"
-   DESTINATION include
+   DESTINATION include/rocclr
    FILES_MATCHING PATTERN "*.h*")
 INSTALL (
    DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/elf"
-   DESTINATION include
+   DESTINATION include/rocclr
    FILES_MATCHING PATTERN "*.h*")
 INSTALL (
    DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/compiler"
-   DESTINATION include
+   DESTINATION include/rocclr
    FILES_MATCHING PATTERN "*.h*")
 INSTALL (
    DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/utils"
-   DESTINATION include
+   DESTINATION include/rocclr
    FILES_MATCHING PATTERN "*.h*")
 INSTALL (
    DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/platform"
-   DESTINATION include
+   DESTINATION include/rocclr
    FILES_MATCHING PATTERN "*.h*")
 INSTALL (
    DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/thread"
-   DESTINATION include
+   DESTINATION include/rocclr
    FILES_MATCHING PATTERN "*.h*")
 INSTALL (
    DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/os"
-   DESTINATION include
+   DESTINATION include/rocclr
    FILES_MATCHING PATTERN "*.h*")
 INSTALL (
    DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/device"
-   DESTINATION include
+   DESTINATION include/rocclr
    FILES_MATCHING PATTERN "*.h*")
